libs/core/src/lib/message-page/message-page.component.ts

```diff
@@ -0,0 +1,128 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ChangeDetectorRef,
+    ElementRef,
+    Input,
+    OnChanges,
+    ViewEncapsulation,
+    OnInit
+} from '@angular/core';
+import { applyCssClass, CssClassBuilder } from '../utils/public_api';
+
+export type MessagePageType = '' | 'filter' | 'search' | 'no-items' | 'error';
+
+@Component({
+    // tslint:disable-next-line:component-selector
```

Comment:
Why disable style check, if it could be configured in angular.json with schematics?